sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala

```diff
@@ -140,8 +140,8 @@ case class CreateDataSourceTableAsSelectCommand(
         return Seq.empty
       }
 
-      saveDataIntoTable(
-        sparkSession, table, table.storage.locationUri, query, mode, tableExists = true)
+      saveDataIntoTable(sparkSession, table, table.storage.locationUri, query, mode,
```

Comment:
we should also explain the behavior in the PR description: When path already exists, we should append to it or overwrite it?